src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,70 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = false;
+    Animal.alive.push(this);
+  }
+
+  isAlive() {
+    return this.health > 0;
+  }
+
+  checkHealth() {
+    if (this.health <= 0) {
```

Comment:
```suggestion
    if (!this.isAlive()) {
```
You have a method to check if animal is alive